gateway/ln-gateway/src/lib.rs

```diff
@@ -134,6 +134,26 @@ pub struct GatewayOpts {
     pub num_route_hints: Option<usize>,
 }
 
+/*
```

Comment:
nit, commented code?

Reply:
Fixed in a later commit , perhaps I should squash those two together